docs/Accessibility.md

```diff
@@ -0,0 +1,34 @@
+# Accessibility
+
+The component library has been built with accessibility in mind.
+
+The components are compliant with Government Design System [GDS](https://design-system.service.gov.uk/patterns/), and meet the [WCAG](https://www.w3.org/TR/WCAG22/) 2.2 Accessibility Guidelines.
+
+## Accessibility validation
+
+Various tools are used to ensure applications built with our components are accessible
+
+-	[Microsoft Accessibility Insights](https://accessibilityinsights.io/): A full manual and automated assessment currently achieves a 91% passing score for all WCAG 2.1 and 2.2 AA criteria.
+-	[Lighthouse Report](https://developer.chrome.com/docs/lighthouse/overview): The report from this tool gives a 100% score.
+-	[Axe DevTools](https://www.deque.com/axe/devtools/): This tool detects no issues.
+
+As well as running automated tooling, we manually test our components as they are being developed.
+
+-	NVDA screenreader testing: Verifying tabbing, navigation, headings, error messages, and submission based on WCAG guidelines.
+- Screen magnifier testing: Verifying magnification at 6x and 8x, label proximity validation, and synchronisation of magnifier and screenreader feedback.
```

Comment:
Mention testing on Windows Magnifier and on desktop and mobile applications using our components.